dotcom-rendering/src/web/components/ArticleHeadline.stories.tsx

```diff
@@ -313,27 +313,42 @@ export const Comment = () => {
 Comment.story = { name: 'Comment' };
 
 export const Analysis = () => {
-	const format = {
+	const themes: [string, ArticleTheme][] = [
+		['News', ArticlePillar.News],
+		['Opinion', ArticlePillar.Opinion],
+		['Sport', ArticlePillar.Sport],
+		['Culture', ArticlePillar.Culture],
+		['Lifestyle', ArticlePillar.Lifestyle],
+	];
+	const format = (theme: ArticleTheme): ArticleFormat => ({
```

Comment:
I know this is (hugely) subjective but are all these types needed? I personally find it makes the code harder to read

Reply:
Yeah you can’t please TS otherwise. Not even with JSDoc types. Do you prefer the following?

```ts
type Fromatter = (theme: ArticleTheme) => ArticleFormat
const format: Fromatter = (theme) => ({ /* … */ });
```